.travis.yml

```diff
@@ -9,3 +9,5 @@ before_install:
     - pip install pylint
 script:
     - pylint bin libfulltext
+after_success:
+    - pip install .
```

Comment:
I suspect `.` is not the best directory specification. Does any of you know if there is an environment variable in travis like `${CURRENT_CHECKOUT_DIR}`?

Reply:
I think this is fine.